csharp/ql/src/Stubs/Stubs.qll

```diff
@@ -122,7 +122,8 @@ abstract private class GeneratedType extends Type, GeneratedElement {
   private string stubComment() {
     result =
       "// Generated from `" + this.getQualifiedName() + "` in `" +
-        concat(this.getALocation().toString(), "; ") + "`\n"
+        concat(Location l | l = this.getALocation() | l.toString(), "; " order by l.toString()) +
```

Comment:
`x order by x` is redundant; at least according to https://codeql.github.com/docs/ql-language-reference/expressions/ that's teh default, so I don't think this particular change does anything.

Reply:
I guess you're right.